pydicom/values.py

```diff
@@ -824,12 +824,8 @@ def convert_value(
     'US': (convert_numbers, 'H'),
     'UT': convert_single_string,
     'UV': (convert_numbers, 'Q'),
-    'OW/OB': convert_OBvalue,  # note OW/OB depends on other items,
-    'OB/OW': convert_OBvalue,  # which we don't know at read time
-    'OW or OB': convert_OBvalue,
```

Comment:
Can these also be removed in the writers and in the exclusions for `DataElement.value`? As far as I can see, these are never set.

Reply:
Done